single_iwyu.py

```diff
@@ -22,7 +22,16 @@ def main(commands: Path, fixer_path: Path, filters: List[Path], specific: str):
         if len(eligible) == 0:
             warn("NO FILE WITH IDENTIFIER FOUND")
         for part in eligible:
-            perform_iwyu(fixer_path, part, filters, current_dir, debug=DEBUG)
+            if not "arch" in part["file"]:
```

Comment:
should you check that arch isn't the first part of the file? Otherwise wouldn't something like linux/farch.c (made up filename) match this?